### PR TITLE
ui: add learn more to alert

### PR DESCRIPTION
**Pull request by maryliag**
Informs ##66987

Epic: None

Add Learn More link to not finalized alert.

Screenshot 2023-06-15 at 2 51 16 PM

Release note: None

### CODE REVIEW

File: pkg/ui/workspaces/db-console/src/redux/alerts.ts [hunk 1]
```diff
@@ -618,7 +618,9 @@ export const upgradeNotFinalizedWarningSelector = createSelector(
     return {
       level: AlertLevel.WARNING,
       title: "Upgrade not finalized.",
-      text: `All nodes are running on version ${nodesVersion}, but the cluster is on version ${clusterVersion}.`,
+      text: `All nodes are running on version ${nodesVersion}, but the cluster is on version ${clusterVersion}. 
+      Features might not be available on this state.`,
```
Comment (rafiss) [inline pkg/ui/workspaces/db-console/src/redux/alerts.ts:622]: super nit: "on this state" -> "in this state"

**Review by rafiss — APPROVED** [review]
lgtm!

**Review by gtr — APPROVED** [review]
should there be a release note? i think the "learn more" link could count as a user-visible change. otherwise, LGTM!

**Review by maryliag — COMMENTED** [review]
not a significant user change in this case, so I will skip the release note

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/104998#-:-NY-g-rq3FUQZbSeXrhE:b3s9x38)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @florence-crl, @kevin-v-ngo, and @rafiss)

___
*[`pkg/ui/workspaces/db-console/src/redux/alerts.ts` line 622 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/104998#gh-1231426289:-NY-g589EFvux6L1pQY7:bb6un1j) ([raw file](https://github.com/cockroachdb/cockroach/blob/8a7b6aa347c52edc24e200e60d19338d9263b363/pkg/ui/workspaces/db-console/src/redux/alerts.ts#L622)):*
<details><summary><i>Previously, rafiss (Rafi Shamim) wrote…</i></summary><blockquote>

super nit: "on this state" -> "in this state"
</blockquote></details>

Done

**Review by kevin-v-ngo — APPROVED** [review]
Thank you!

**Pull request merged**